src/docfx/SubCommands/BuildCommand.cs

```diff
@@ -245,9 +245,32 @@ private static void MergeOptionsToConfig(BuildCommandOptions options, BuildJsonC
             {
                 config.IntermediateFolder = options.IntermediateFolder;
             }
+            if (options.GlobalMetadataFilePaths != null && !options.GlobalMetadataFilePaths.Any())
+            {
+                if (!config.GlobalMetadataFilePaths.Any())
+                {
+                    Logger.LogWarning($"globalMetadata config file from command line overrides it from docfx config file.");
```

Comment:
>globalMetadata config file from command line overrides it from docfx config file. [](start = 40, length = 81)

no need to warn? From the code below it's combination of command line and config file.